.github/workflows/test.yml

```diff
@@ -26,14 +26,17 @@ jobs:
           cache: "pnpm"
       - name: Install Dependencies
         run: pnpm install --frozen-lockfile
-      - name: Run tests
-        run: pnpm run test:docker
-      - name: Upload Playwright Report
-        uses: actions/upload-artifact@v4
-        if: always()
-        with:
-          name: playwright-report-${{ github.sha }}-${{matrix.os}}-${{ matrix.node-version }}
-          path: ./test-artifacts/playwright
```

Comment:
We still want to run the tests in docker, just not run the playwright step. I think you can just remove the  test:playwright bit from the test script in package.json for now.